Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM tomcat:8-jdk8-openjdk
+
+ENV PATH="/bin/liquibase:${PATH}" 
+ENV LIQUIBASE_HOME="/bin/liquibase"
```

Comment:
I believe `/opt/liquibase` would be a more unix-y place to install liquibase.
Also, we should install `lpm` into `/usr/local/bin` and create a symlink for `liquibase` in there as well. This will alleviate the need to modify `PATH`.
`LIQUIBASE_HOME` does not seem to be required and is not set in the [official liquibase Docker image](https://github.com/liquibase/docker/blob/main/Dockerfile).